kiosk/styles/layout/HeroSection.css

```diff
@@ -35,13 +40,11 @@
       }
 
       .slogan {
-        @apply text-white uppercase pt-5 pb-2 font-normal;
-
-        font-size: 1em;
+        @apply text-white uppercase pt-5 pb-2 font-normal text-2xl font-bold;
```

Comment:
You have both `font-normal` and `font-bold` in the same CSS definition. Stylelint should have warned you about that in VSCode if you have the Stylelint extension installed. It is not an error so it is not reported by running `npm run lint`.